_posts/2016-04-11-growing-developers-at-greenfield.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Growing Developers at Greenfield"
+description: "Advice for junior developers getting started in their careers"
+category:
+tags: [lessons, advice, career]
+author: tom_chen
+---
+{% include JB/setup %}
+
+![tree]({{ BASE_PATH
+}}/assets/images/growing-developers-at-greenfield/tree.jpeg)
+
+Before I was a developer at Greenfield, I spent several years working in
```

Comment:
`.. taught in Academia`? 'working in academia' could be anyone. might want to say more about your background here (I was a Professor/Researcher/etc.)